baseclasses/pyAero_solver.py

```diff
@@ -26,13 +26,13 @@ class AeroSolver(BaseSolver):
     Abstract Class for Aerodynamic Solver Object
     """
 
-    def __init__(self, name, category={}, def_options={}, informs={}, options={}):
+    def __init__(self, name, *args, **kwargs):
```

Comment:
Not sure we should change this here since it may be helpful to know what this child class expects in the initialization. Same below.
We should discuss this in person.